transifex_catalogs/en.json

```diff
@@ -46,10 +46,10 @@
   "downloads_category_core_software": "Core software",
   "downloads_category_third_party_wallets": "3rd party wallets",
   "downloads_category_hardware_wallets": "Hardware wallets",
-  "downloads_app_decrediton": "Graphical UI wallet for Windows, macOS and Linux.",
-  "downloads_app_dcrdex": "Multi-coin light wallet with DEX support. No staking or advanced wallet features.",
-  "downloads_app_cryptopower": "Multi-coin light GUI wallet for DCR, BTC and LTC. Built purely in Go.",
-  "downloads_app_commandline": "A cross-platform, automatic installer/updater for the command-line applications.",
+  "downloads_app_decrediton": "Graphical UI wallet with all Decred features: staking, voting, Lightning Network, Trezor, DEX trading, and more. Runs on Windows, macOS and Linux.",
+  "downloads_app_dcrdex": "Multi-coin light wallet with peer-to-peer trustless trading and basic wallet features for 9 assets. DCR staking is coming. See the website for an introduction.",
+  "downloads_app_cryptopower": "Multi-coin light GUI wallet for DCR, BTC and LTC. Built purely in Go. Windows, macOS, Linux and FreeBSD.",
```

Comment:
if we would be adding the platforms, we could include mobile also

Reply:
I'm balancing between helping users navigate all these choices and adding too much detail. Adding platform names seemed important, esp. if they are part of what differentiates the product.

How about "Android and iOS is coming"?

Reply:
Added "Android and iOS are coming." at the end for now. Can adjust further.